Fw/Python/src/fprime/util/build_helper.py

```diff
@@ -71,21 +67,9 @@ def get_build(
         build meeting the specifications on build type
     """
     build_types = target.build_types if target is not None else BuildType
```

Comment:
Wouldn't this need to be build_type not build_types.  You reduced the targets to a single item.